### PR TITLE
Add concrete powder and other misc. blocks

### PR DESCRIPTION
**Pull request by jspanos71**
Adding concrete powder and fixing glazed terracotta / glass panes

### CODE REVIEW

File: overviewer_core/world.py [hunk 1]
```diff
@@ -468,7 +467,6 @@ def __init__(self, regiondir, rel):
             'minecraft:chiseled_stone_bricks': (98, 3),
             'minecraft:infested_chiseled_stone_bricks': (98, 3),
             'minecraft:brown_mushroom_block': (99, 0),
-            'minecraft:glass_pane': (20, 0), # wrong
```
Comment (gmcnew) [inline overviewer_core/world.py:471]: I don't believe this should be removed -- glass does not have to have a color. https://minecraft.gamepedia.com/Glass_Pane

File: overviewer_core/world.py [hunk 2]
```diff
@@ -396,8 +396,6 @@ def __init__(self, regiondir, rel):
             'minecraft:pink_tulip': (38, 7),
             'minecraft:oxeye_daisy': (38, 8),
             'minecraft:dandelion': (38, 9),
-            'minecraft:brown_mushroom': (39, 0),
-            'minecraft:red_mushroom': (40, 0),
```
Comment (gmcnew) [inline overviewer_core/world.py]: Why are these being removed?

File: overviewer_core/world.py [hunk 3]
```diff
@@ -526,14 +524,31 @@ def __init__(self, regiondir, rel):
...
+            'minecraft:pink_terracotta': (159, 6),
+            'minecraft:gray_terracotta': (159, 7),
+            'minecraft:silver_terracotta': (159, 8),
+            'minecraft:cyan_terracotta': (159, 9),
+            'minecraft:purple_terracotta': (159, 10),
+            'minecraft:blue_terracotta': (159, 11),
+            'minecraft:brown_terracotta': (159, 12),
+            'minecraft:green_terracotta': (159, 13),
+            'minecraft:red_terracotta': (159, 14),
+            'minecraft:black_terracotta': (159, 15),
```
Comment (gmcnew) [inline overviewer_core/world.py:544]: Can we continue handling these in the "for each color" loop toward the end of this function?

File: overviewer_core/world.py [hunk 4]
```diff
@@ -606,61 +621,28 @@ def __init__(self, regiondir, rel):
...
+            'minecraft:pink_glazed_terracotta': (241, 0),
+            'minecraft:gray_glazed_terracotta': (242, 0),
+            'minecraft:silver_glazed_terracotta': (243, 0),
+            'minecraft:cyan_glazed_terracotta': (244, 0),
+            'minecraft:purple_glazed_terracotta': (245, 0),
+            'minecraft:blue_glazed_terracotta': (246, 0),
+            'minecraft:brown_glazed_terracotta': (247, 0),
+            'minecraft:green_glazed_terracotta': (248, 0),
+            'minecraft:red_glazed_terracotta': (249, 0),
+            'minecraft:black_glazed_terracotta': (250, 0),
```
Comment (gmcnew) [inline overviewer_core/world.py:641]: Can we handle these in the "for each color" block?
`self._blockmap['minecraft:%s_glazed_terracotta' % colors[i]] = (235 + i, 0)`

**Review by gmcnew — CHANGES_REQUESTED** [review]
Thank you very much for this! I left some comments for some things that would be nice to fix.

File: overviewer_core/world.py (hunk 1 above)
Comment (IncredibleHolg) [reply overviewer_core/world.py:471]: 20,0 for glass_pane  is the wrong value, this is the value of minecraft:glass.

minecraft:glass_pane': (102, 0), is the corect value. 

Btw. the stained glass values are wrong (241 which is terracotta) the value should be 95

```python
            'minecraft:white_stained_glass': (95, 0),
            'minecraft:orange_stained_glass': (95, 1),
            'minecraft:magenta_stained_glass': (95, 2),
            'minecraft:light_blue_stained_glass': (95, 3),
            'minecraft:yellow_stained_glass': (95, 4),
            'minecraft:lime_stained_glass': (95, 5),
            'minecraft:pink_stained_glass': (95, 6),
            'minecraft:gray_stained_glass': (95, 7),
            'minecraft:light_gray_stained_glass': (95, 8),
            'minecraft:cyan_stained_glass': (95, 9),
            'minecraft:purple_stained_glass': (95, 10),
            'minecraft:blue_stained_glass': (95, 11),
            'minecraft:brown_stained_glass': (95, 12),
            'minecraft:green_stained_glass': (95, 13),
            'minecraft:red_stained_glass': (95, 14),
            'minecraft:black_stained_glass': (95, 15),
```

File: overviewer_core/world.py (hunk 3 above)
Comment (jspanos71) [reply overviewer_core/world.py:544]: The problem is that %s_terracotta matches both glazed and regular terracotta.  We could keep the glazed in as it's more specific, but this one would require a more complicated matching pattern.  Easier and more explicit to just use this mapping.

File: overviewer_core/world.py (hunk 4 above)
Comment (jspanos71) [reply overviewer_core/world.py:641]: Yes, we could handle this one in a loop.  I took it out just to be a little clearer and not be tempted into the the trap mentioned here: https://github.com/gmcnew/Minecraft-Overviewer/pull/4/files/bd78d7cd8352a45aaa38fecbfb121c20597ed6e4#r212585889

File: overviewer_core/world.py (hunk 1 above)
Comment (jspanos71) [reply overviewer_core/world.py:471]: The glass_pane mapping was in there twice, the first one is this one and was wrong.  Removing it left only the correct one (102).

**Comment by jspanos71** [comment]
Restored the mushroom items and fixed where visual studio added tabs instead of spaces.  I would squash it myself, but I don't know how and am pressed for time at the moment.

**Review by gmcnew — APPROVED** [review]

**Pull request merged**